README.md

```diff
@@ -262,6 +264,65 @@ func main() {
 }
 ```
 
+### Send a mail through a proxy
+``` go
+package main
+
+import (
+	"github.com/mailjet/mailjet-apiv3-go"
+	"github.com/mailjet/mailjet-apiv3-go/resources"
+
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+)
+
+// Set the http client with the given proxy url
+func setupProxy(url string) *http.Client {
```

Comment:
This is merely a simple example
This helper method will not be enough to support any kind of proxy configuration
I just did this function to bring more visibility to the use case